platforms/corda-ent/charts/idman/templates/deployment.yaml

```diff
@@ -83,80 +60,78 @@ spec:
                 fi
               }
 
-              # setting up env to get secrets from vault
-              echo "Getting secrets from Vault Server"
+              # Setting up the environment to get secrets/certificates from Vault
+              echo "Getting certificates/secrets from Vault Server"
               KUBE_SA_TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
               VAULT_TOKEN=$(curl -sS --request POST ${VAULT_ADDR}/v1/auth/${KUBERNETES_AUTH_PATH}/login -H "Content-Type: application/json" -d '{"role":"vault-role","jwt":"'"${KUBE_SA_TOKEN}"'"}' | jq -r 'if .errors then . else .auth.client_token end')
               validateVaultResponse 'vault login token' "${VAULT_TOKEN}"
-              echo "logged into vault"
+              echo "Logged into Vault"
               # Creating dirs for storing certificates
               mkdir -p ${MOUNT_PATH}/idman;
               mkdir -p ${MOUNT_PATH}/root;
               mkdir -p ${MOUNT_PATH}/crl-files;
 
               # Fetching ssl-idman certificates from vault
               COUNTER=1
-              while [ "$COUNTER" -lt {{ $.Values.healthcheck.readinessthreshold }} ]
+              while [ "$COUNTER" -lt {{ $.Values.healthCheck.readinessThreshold }} ]
```

Comment:
in signer u have changed the the variable to vault retries;  u can make it consistent and more meaning for here as well